.travis.yml

```diff
@@ -1,15 +1,16 @@
 language: php
 
 php:
-    - 5.6
-    - 7.0
     - 7.1
     - 7.2
-    - hhvm
+    - 7.3
```

Comment:
Would add PHP 7.4 while you're at it

Reply:
i tried adding 7.4, almost all test failed, but i can add this 🤷‍♂

Reply:
Seems sensible to tackle 7.4 in another PR if its failing hard

Reply:
It was likely failing on PHP 7.4 because of PHPUnit 5, which I've upgraded in a separate PR

Reply:
I'm going to merge + rebase https://github.com/usemarkup/AddressingBundle/pull/23#pullrequestreview-350050474